open-media-match/omm_config.py

```diff
@@ -1,5 +1,5 @@
 """
-A 
+A
```

Comment:
This file is still specific to the .devcontainer postgres instance, which is why I had it in .devcontainer

Reply:
Moved back and created a new omm_config for the production config.